src/GeositeFramework/Models/Geosite.cs

```diff
@@ -30,6 +30,16 @@ public class Link
             public bool Popup;
 
             /// <summary>
+            /// Should this link open a launchpad
+            /// </summary>
+            public bool Launchpad;
```

Comment:
For simplicity, I think we could just get away with the int? `LaunchpadId`.  Since it's nullable, that could be our check.  Reduces the number of attributes the region.json link config would need.